app/Jobs/CreateSoftEdgePostInRogue.php

```diff
@@ -70,8 +70,8 @@ private function extractDetails($email)
         return json_encode([
             'email_timestamp' => $email['email_timestamp'],
             'campaign_target_name' => $email['campaign_target_name'],
-            'campaign_target_title' => $email['campaign_target_title'],
-            'campaign_target_district' => $email['campaign_target_district'],
+            'campaign_target_title' => $email['campaign_target_title'] ? $email['campaign_target_title'] : null,
+            'campaign_target_district' => $email['campaign_target_district'] ? $email['campaign_target_district'] : null,
```

Comment:
Shouldn't `$email['campaign_target_title']` already return `null` if that array key isn't set?

Reply:
oh you're right! Ok going to close this. I saw something weird on QA but just tested again and all is working 👍